src/plugins/data_views/server/rest_api_routes/runtime_fields/response_formatter.ts

```diff
@@ -15,7 +16,11 @@ interface ResponseFormatterArgs {
   dataView: DataView;
 }
 
-export const responseFormatter = ({ serviceKey, fields, dataView }: ResponseFormatterArgs) => {
+export const responseFormatter = ({
+  serviceKey,
+  fields,
+  dataView,
```

Comment:
What do we want  `dataView` to be interpreted as? Is it a Kibana `DataView` or how to "view the data"?

Reply:
The only change here was to more narrowly define the return type of this function to `HttpResponseOptions`. I believe the intention is to refer to the new "Data view" as opposed to the legacy "Index pattern".